packages/dapp/src/components/Widget/Widget.tsx

```diff
@@ -162,9 +163,11 @@ export function Widget({ starterVariant }) {
   ]);
 
   return (
-    <LiFiWidget
-      integrator={import.meta.env.VITE_WIDGET_INTEGRATOR as string}
-      config={widgetConfig}
-    />
+    <Box className="widget-wrapper">
+      <LiFiWidget
+        integrator={import.meta.env.VITE_WIDGET_INTEGRATOR as string}
```

Comment:
@chybisov Thought about parsing a className here? Wdyt?

Reply:
It needs to be apparent where this comes from, and why not use a styled component here?

Reply:
It´s been a hazzle, the hover effect´s are triggered on parent components, resulting in changing styles on that given component. This is why I am passing a className which can be used on those parent styles.